Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement/CortexAttackSurfaceManagement.yml

```diff
@@ -418,10 +418,35 @@ script:
     - contextPath: ASM.RemediationScan.result
       description: Result of the Remediation Confirmation Scan.
       type: string
-  dockerimage: demisto/python3:3.10.13.75921
+  - arguments:
```

Comment:
Should any of these be mandatory?  I believe at least `attack_surface_rule_id` should be

Reply:
@johnnywilkes, we can still do something like this: `!asm-get-attack-surface-rule enabled_status=on` and it will get us all the enabled rules. In this case, we don't provide attack_surface_rule_id. Hence, I made all as optional.